### ISSUE TITLE
Complex content in scrollable panel overlap the panel title 

### ISSUE THREAD

**Issue by roberchi** [microsoft/fluentui#20807]
I created a panel like this:

```
<Panel
        type={PanelType.custom}
        isOpen={isOpen}
        onDismiss={dismissPanel}
        onRenderFooterContent={onRenderFooterContent}
        isFooterAtBottom={true}
        closeButtonAriaLabel="Close"
        headerText={"Settings panel"}
        >
          {settings}
</Panel>
```
in {setting} if I've got simple like this that is longer than the panel height, if is scroll it it works as expected.
```
<Label>item 1</Label>
<Label>item 2</Label>
<Label>item 3</Label>
... etc.
```
but if in the content {settings} I've some other components like this:
```<Label>
      Containers <IconButton iconProps={{ iconName: "add" }}></IconButton>
    </Label>
    <Stack key={index}>
          <Stack.Item>
            <TextField
              label="Name"
              required
            ></TextField>
            <IconButton iconProps={{ iconName: "delete" }}></IconButton>
          </Stack.Item>
          <Stack.Item>
            <TextField
              label="Url"
              required        
              multiline
              rows={3}
            ></TextField>
          </Stack.Item>
        </Stack>
... and repeating...
```
when I scroll it than it overlap the title has shown above:

![image](https://user-images.githubusercontent.com/12184405/143659255-5a79545d-8adb-4516-92a9-f836c14062f0.png)

Here my sample to reproduce it:
https://codepen.io/robertoch/pen/zYdgbOK

if I use the old version of the library  **office-ui-fabric-react** instead of **@fluentui/react** this behavior doesn't happen.

**Comment by kollster** [comment]
We have the same issue with panel content flowing over the header.

**Comment by nanivijay** [comment]
+1

**Comment by gouttierre** [comment]
@roberchi - Thanks for filing this issue with us. To set expectations, the developers will review the issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by TristanWatanabe** [comment]
Closing this as it was fixed by #20806
